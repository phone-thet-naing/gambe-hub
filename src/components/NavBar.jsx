import { Button, ButtonSpinner, HStack, Image } from "@chakra-ui/react"
import logo from '../assets/m.jpg'
import ColorModeSwitch from "./ColorModeSwitch"

function NavBar() {
  return (
    <HStack justifyContent='space-between' padding='0.25rem 0.5rem'>
        <Image src={logo} boxSize='55px' borderRadius='full' boxShadow='base' />
        <ColorModeSwitch />
    </HStack>
  )
}

export default NavBar