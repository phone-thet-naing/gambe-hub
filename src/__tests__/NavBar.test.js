import NavBar from '../components/Navbar'
import renderer from 'react-test-renderer'

jest.mock('HStack', () => 'HStack')
jest.mock('Image', () => 'Image')

it('navbar renders correctly', () => {
    const navbar = renderer.create(<NavBar />).toJSON();
    expect(navbar).toMatchSnapshot();
})