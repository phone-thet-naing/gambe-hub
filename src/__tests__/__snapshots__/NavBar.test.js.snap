// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`navbar renders correctly 1`] = `
<h1>
  NavBar
</h1>
`;
